base/ltfinal.dtx

```diff
@@ -1325,7 +1325,25 @@
 %    \end{macrocode}
 %  \end{macro}
 %
+%   To allow package and class author to support for document links
+%   we provide also the new interface commands of the hyperref package
+%   for the creation of targets.
 %
+%  \begin{macro}{\MakeLinkTarget}
+%  \begin{macro}{\LinkTargetOn}
+%  \begin{macro}{\LinkTargetOff}
+%  \begin{macro}{\NextLinkTarget}
+% \changes{v2.2u}{2022/05/10}{Default definition for targets added}
+%    \begin{macrocode}
+\NewDocumentCommand\MakeLinkTarget{sO{}m}{}
+\NewDocumentCommand\LinkTargetOn{}{}
+\NewDocumentCommand\LinkTargetOff{}{}
+\NewDocumentCommand\NextLinkTarget{m}{}
```

Comment:
As mentioned by @zauguin  those should probably do more than nothing but insert a whatsit depending on the engine so that spacing is identical whether or not real actions happen.

Reply:
Only the first needs imho a content. In vertical mode a special can be used, but in hmode hyperref inserts a box and surrounds it with `\Hy@SaveSpaceFactor`  and`\Hy@RestoreSpaceFactor`.  How would one  reproduce that at best?

Reply:
well you can mimic the mechanism if that's what it should do, because then this behavior would bessentially part of the command not part of what hyperref does with it (perhaps, dependes on where how it could/should be used in other places, but my guess is that this is *part of the command* in reality)